src/plugin.js

```diff
@@ -10,10 +12,13 @@ import OrbitOrientationContols from './orbit-orientation-controls.js';
 import * as utils from './utils';
 import CanvasPlayerControls from './canvas-player-controls';
 import OmnitoneController from './omnitone-controller';
+import { VRButton } from '../vendor/three/VRButton.js';
 
 // import controls so they get regisetered with videojs
 import './cardboard-button';
 import './big-vr-play-button';
+import {XRControllerModelFactory} from '../webxr/libs/three/jsm/XRControllerModelFactory';
+import {BoxLineGeometry} from '../webxr/libs/three/jsm/BoxLineGeometry';
```

Comment:
tests are failing because these files don't exist. Are these supposed to be in the webxr polyfill?

Reply:
Oh - I'll do a clean checkout to be sure

Reply:
My bad - I had a local copy of three.js I was referencing
I've updated to reference the node_modules three.js example as that what I copied